subgraph/config.py

```diff
@@ -15,9 +15,9 @@
         f"tokens-{Network.Arbitrum}": "https://api.thegraph.com/subgraphs/name/darruma/badger-tokens-arbitrum",
         "fuse": "https://api.thegraph.com/subgraphs/name/darruma/fuse-subgraph-badger",
         "thegraph": "https://api.thegraph.com/index-node/graphql",
-        "nfts": "https://api.thegraph.com/subgraphs/name/darruma/badger-nfts"
+        "nfts": "https://api.thegraph.com/subgraphs/name/darruma/badger-nfts",
     },
     Environment.Staging: {
         Network.Ethereum: "https://api.thegraph.com/subgraphs/name/badger-finance/badger-dao-setts",
```

Comment:
I think we should revert this for now, until we're ready to do complete testing.  Using this in staging can produce unpredictable results from testing compared to what's in production.

Reply:
There is no difference between the urls here, the linter just added a comma.

Reply:
What I meant was `Environment.Staging` is returning a different subgraph than `Environment.Production`.  If we're using a separate subgraph for testing this should be very explicitly used only when we're testing new subgraph upgrades, and definitely not accidentally used in integration tests where the two subgraphs may produce different results.

Reply:
Ok, removed the staging subgraph